esmvalcore/cmor/_fixes/cmip6/mcm_ua_1_0.py

```diff
@@ -106,25 +130,22 @@ def fix_metadata(self, cubes):
         return [cube]
 
 
-class Omon(Fix):
-    """Fixes for ocean variables."""
+class Uas(Fix):
+    """Fixes for uas."""
 
     def fix_metadata(self, cubes):
-        """Fix ocean depth coordinate.
+        """Add height (10m) coordinate.
 
         Parameters
         ----------
-        cubes: iris CubeList
-            List of cubes to fix
+        cubes : iris.cube.CubeList
+            Cubes to fix.
 
         Returns
         -------
-        iris.cube.CubeList
+        iris.cube.Cube
```

Comment:
you're technically returning a list of one cube, but still a list so the API is incorrect, maybe change with `list \n list of [cube]`